nf/modules/local/sentieon/dnascope/main.nf

```diff
@@ -0,0 +1,35 @@
+process SENTIEON_DNASCOPE {
+    label 'C32'
+    container "pgc-images.sbgenomics.com/hdchen/sentieon:202308.03"
+
+    input:
+    tuple path(alignment), path(align_index)
+    tuple path(reference), path(fai)
+    path(interval)
+    val(ploidy)
+    tuple path(dbsnp), path(dbsnp_index)
+
+
+    output:
+    tuple path('*vcf.gz'), path('*vcf.gz.tbi'), emit: output_vcf
+    script:
+    def license_export = task.ext.export_args ?: ''
+    def dbsnp_flag = dbsnp ? "-d $dbsnp" : ''
+    def algo_ext_args = task.ext.algo_args ?: ''
+    """
+    $license_export \\
```

Comment:
You can set environmental variables through input: https://www.nextflow.io/docs/latest/process.html#input-environment-variables-env

Reply:
I guess the problem here is how do we handle the file...